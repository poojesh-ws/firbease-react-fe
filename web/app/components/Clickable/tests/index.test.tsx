/**
 *
 * Tests for Clickable
 *
 */

import React from 'react';

import { renderWithIntl } from '@utils/testUtils';
import Clickable from '../index';

describe('<Clickable /> component tests', () => {
  let clickSpy: jest.Mock;
  beforeAll(() => {
    clickSpy = jest.fn();
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderWithIntl(<Clickable textId="launches_list" onClick={clickSpy} />);
    expect(baseElement).toMatchSnapshot();
  });
});
