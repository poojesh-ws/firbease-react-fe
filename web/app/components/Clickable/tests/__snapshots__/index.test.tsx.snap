// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Clickable /> component tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="Clickable__StyledClickable-sc-1vn3tbu-0 gFYhKE"
      data-testid="clickable"
    >
      <p
        class="T__StyledText-znbtqz-0 hGCwnP"
        data-testid="t"
      >
        List of launches
      </p>
    </div>
  </div>
</body>
`;
